src/meshapi/urls.py

```diff
@@ -24,10 +23,8 @@
     path("query/buildings/", views.QueryBuilding.as_view(), name="meshapi-v1-query-building"),
     path("query/members/", views.QueryMember.as_view(), name="meshapi-v1-query-member"),
     path("query/installs/", views.QueryInstall.as_view(), name="meshapi-v1-query-install"),
-    path("mapdata/installs/", views.MapDataInstallList.as_view(), name="meshapi-v1-map-data-installs"),
+    path("mapdata/nodes/", views.MapDataNodeList.as_view(), name="meshapi-v1-map-data-installs"),
```

Comment:
Not super jazzed about canonizing a "nodes" route. I am assuming this is purely for backwards compatibility with the map, and any new map wouldn't do this.

I would argue for a "DISCLAIMER THIS IS FUCKING RADIOACTIVE DON'T FUCKING USE THIS" docstring somewhere.

Reply:
The whole `/mapdata` path is purely for backwards compatibility. I renamed it to `nodes` to more accurately reflect the hacky "installs get overwritten with NNs" behavior. It doesn't just show you installs, it shows you `nodes` with all of the historical baggage that term brings

Reply:
But heard, I will add a disclaimer on each endpoint

Reply:
<img width="1451" alt="Screenshot 2024-03-02 at 13 01 59" src="https://github.com/WillNilges/meshdb/assets/3138937/cea17405-a610-4c09-b74b-9c2540b0777e">
This feels... excessive to the point of being disrespectful. But if this is what you want we can do it

Reply:
OK then, perhaps just a 
```
(Warning: This endpoint is a legacy format and may be deprecated/removed in the future)
```